tests/test_gradient.py

```diff
@@ -1 +1,11 @@
-# Last names: L, M
\ No newline at end of file
+# Last names: L, M
+
+from minecraft_gen import gradient
+import numpy as np
+
+def test_gradient():
+    dummy_im_smooth = np.random.default_rng().random(100)[np.newaxis]
```

Comment:
```suggestion
    gen = np.random.default_rng(42)
    dummy_im_smooth = gen.random(100)[np.newaxis]
```

You should set the seed, so the test is repeatable. If you don't set a seed, this is a property-based test, and it's better to use a framework to help with it (Hypothesis, for example).